### PR TITLE
Db/ttgo tq support

### PR DESCRIPTION
**Pull request by danielbernalb**
TTGO_TQ board support: 
* Added support to board TTGO TQ with a 128x32 OLED screen
* Added battery level number indicator from chip IP5306. You must solder:
PIN2 of IP5306 with PIN27 of TTGO_TQ
PIN3 of IP5306 with PIN14 of TTGO_TQ
PIN 4 in the picture is solder to TTGO_TQ but is not necessary. 

IMG_20190814_151554

Option for TTGO18650 board:
* Pins for USART

### CODE REVIEW

File: lib/gui-utils/src/GUIUtils.hpp [hunk 1]
```diff
@@ -43,7 +43,7 @@ class GUIUtils
 
   void displaySensorAvarage(int avarage);
 
-  void displaySensorData(int pm25, int pm10);
+  void displaySensorData(int pm25, int pm10, int chargeLevel);
```
Comment (hpsaturn) [inline lib/gui-utils/src/GUIUtils.hpp]: is possible maybe that we need a different prototype of this method, also we need using preprocessor directives in the contents of this method

File: platformio.ini [hunk 2]
```diff
@@ -35,6 +35,7 @@ lib_deps = ${common_env_data.lib_deps}
 build_flags =
     ${common_env_data.build_flags}
     -D D1MINI=1
+    -D TTGO_TQ=0
```
Comment (hpsaturn) [inline platformio.ini]: with #IFNDEF directive you dont need XXFLAG=0

File: src/main.cpp [hunk 3]
```diff
@@ -38,6 +38,10 @@
 U8G2_SSD1306_128X64_NONAME_F_SW_I2C u8g2(U8G2_R0, 4, 5, U8X8_PIN_NONE);
 #elif HELTEC // display via i2c for Heltec board
 U8G2_SSD1306_128X64_NONAME_F_SW_I2C u8g2(U8G2_R0, 15, 4, 16);
+#elif TTGO18650 // display via i2c for TTGO18650
```
Comment (hpsaturn) [inline src/main.cpp]: these directives is the right form for enable/disable code. It's ok.

File: lib/gui-utils/src/GUIUtils.cpp [hunk 4]
```diff
@@ -44,26 +44,59 @@ void GUIUtils::showProgress(unsigned int progress, unsigned int total){
 
 void GUIUtils::welcomeAddMessage(String msg){
   u8g2.setFont(u8g2_font_4x6_tf);
+if (TTGO_TQ == 1) {
```
[the remark this answers is not in the document]
Comment (danielbernalb) [reply lib/gui-utils/src/GUIUtils.cpp]: Resolved with preprocessor directives

File: src/main.cpp (hunk 3 above)
Comment (danielbernalb) [reply src/main.cpp]: Good

**Pull request merged**